Bubble_Sort

```diff
@@ -0,0 +1,42 @@
+# Bubble Sort Algorithm ASC
+N = 3
```

Comment:
using N is redundant - need to use len(a) in lines 6 and 7
in cases of a = ["test", "My", "words" , "word4", "word5"] the only first 3 elements will be sorted